codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodecBuilder.java

```diff
@@ -149,6 +150,18 @@ public Http2MultiplexCodecBuilder initialHuffmanDecodeCapacity(int initialHuffma
         return super.initialHuffmanDecodeCapacity(initialHuffmanDecodeCapacity);
     }
 
+    /**
+     * Determines whether a {@link Http2SettingsFrame} should be propagated in the parent channel pipeline. Default is
+     * false and SETTINGS frames will be processed internally by {@link Http2MultiplexCodec}.
```

Comment:
nit: false -> `{@code false}`

Reply:
Also maybe make it clear that even if `true` there will also processed internally.